automata/symbol/graph/symbol_graph.py

```diff
@@ -14,6 +16,10 @@
 
 logger = logging.getLogger(__name__)
 
+data_root_path = os.environ.get("DATA_ROOT_PATH")
```

Comment:
Let's get `DATA_ROOT_PATH` into the .env.example. Also, rather than having `DATA_ROOT_PATH` ever fail, we should just set a reasonable default in the /config/__init__.py

Reply:
e.g. this is what we currently have

```python
load_dotenv()

# Define environment variables
OPENAI_API_KEY = os.getenv("OPENAI_API_KEY", "")
GITHUB_API_KEY = os.getenv("GITHUB_API_KEY", "")
CONVERSATION_DB_PATH = os.getenv(
    "CONVERSATION_DB_PATH", os.path.join("..", "conversation_db.sqlite3")
)
EVAL_DB_PATH = os.getenv("EVAL_DB_PATH", os.path.join("..", "eval_db.sqlite3"))
TASK_DB_PATH = os.getenv("TASK_DB_PATH", os.path.join("..", "task_db.sqlite3"))
TASK_OUTPUT_PATH = os.getenv(
    "TASKS_OUTPUT_PATH", os.path.join("..", "local_tasks")
)
REPOSITORY_NAME = os.getenv("REPOSITORY_NAME", "emrgnt-cmplxty/Automata")
MAX_WORKERS = int(os.getenv("MAX_WORKERS", 8))

```